samples/frameworks/electron/ts/README.md

```diff
@@ -0,0 +1,77 @@
+# Azure SDK samples for Electron (TypeScript)
+
+This sample application shows how to use the TypeScript client libraries for Azure in some common scenarios.
+
+In this sample, we build a simple [Electron][electron] application and integrating with various Azure services.
+
+- Integration with Azure Service Bus to support a single instance publishing and receiving Service Bus messages.
+- Integration with Azure Storage Blob for persisting notes.
+
+## Prerequisites
+
+The samples are compatible with Node.js >= 8.0.0.
+
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation][typescript].
+
+You need [an Azure subscription][freesub] and the following resources created to run this sample:
+
+- An Azure ServiceBus namespace and queue. Please refer to the [ServiceBus documentation][servicebus] for additional information on ServiceBus.
+- An Azure Storage Blob container. Please refer to the [Storage Blob documentation][storageblob] for additional information on Azure Storage Blob. This file will be fetched from Azure Storage Blob and displayed on the screen.
+- Finally, you'll need a way to authenticate the application with Azure. Please refer to the [@azure/identity][identity] package for information on authentication. The instructions below will walk you through the necessary steps.
+
+To quickly create the needed resources in Azure and to receive the necessary environment variables for them, you can deploy our sample template by clicking:
+
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsamples%2Fframeworks%2Felectron%2Farm-template.json)
```

Comment:
Since the arm template isn't merged to master yet you can use this link for now:

[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2F1964a72677c0eaa7746568010b5e506a0a5321d6%2Fsamples%2Fframeworks%2Felectron%2Farm-template.json)